src/main/java/be/acara/frontend/model/EventWithoutImage.java

```diff
@@ -0,0 +1,45 @@
+package be.acara.frontend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.Future;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EventWithoutImage {
```

Comment:
Wouldn't it be better if we extended event from this class? This could potentially scrap all the duplicate code.